### ISSUE TITLE
Subscribed calendar not showing with JS error in the console

### ISSUE THREAD

**Issue by albatros69** [nextcloud/calendar#1896]
### Steps to reproduce
I'm subscribed to a seemingly genuine ICS calendar, which stopped being displayed after the last upgrade. It was displaying correctly in prior version of NC calendar, and still working in other calendar applications (like Lightning).

The calendar is a public one, so I have attached it to the issue.
[Calendrier_CB.ics.txt](https://github.com/nextcloud/calendar/files/4108412/Calendrier_CB.ics.txt)

### Expected behaviour
Show the calendar as expected

### Actual behaviour
No event is shown and the following error appears three times in the browser console:
[ERROR] calendar: too much recursion

### Calendar app
**Calendar app version:** (see apps admin page, e.g. 2.0.1)
2.0.1

**CalDAV-clients used:** (Thunderbird Lightning, DAVx5, Evolution, macOS Calendar, etc)

### Client configuration
All the browsers I tried show the same behavior (Firefox, Firefox ESR, Edge), on Linux and Windows 

### Server configuration

**Operating system**: (e.g. Debian 8)
Debian 10
**Web server:** (e.g. Apache, Nginx,...)
nginx/1.14.2
**Database:** (e.g. MariaDB, SQLite or PostgreSQL)
PostgreSQL 11.5 (Debian 11.5-1+deb10u1)
**PHP version:** (e.g. 7.0.3)
Version : 7.3.11
**Nextcloud Version:** (see admin page, e.g. 17.0.2)
17.0.2
**Updated from an older installed version or fresh install:**
Updated from older version

**List of activated apps:**

Apps | Version
-- | --
Accessibility | 1.3.0
Activity | 2.10.1
Brute-force settings | 1.5.0
Calendar | 2.0.1
Collaborative tags | 1.7.0
Comments | 1.7.0
Contacts | 3.1.8
Deleted files | 1.7.0
Federation | 1.7.0
File sharing | 1.9.0
First run wizard | 2.6.0
Gallery | 18.4.0
Group folders | 5.0.5
Keeweb | 0.6.1
LDAP user and group backend | 1.7.0
Log Reader | 2.2.0
Maps | 0.1.4
Monitoring | 1.7.0
News | 14.1.2
Nextcloud announcements | 1.6.0
Notes | 3.1.1
Notifications | 2.5.0
Password policy | 1.7.0
PDF viewer | 1.6.0
Polls | 1.0.0
Privacy | 1.1.0
Recommendations | 0.5.0
Right click | 0.15.1
Share by mail | 1.7.0
Support | 1.0.1
Text | 1.1.1
Theming | 1.8.0
Two-Factor TOTP Provider | 4.1.2
Two-Factor U2F | 5.0.2
Update notification | 1.7.0
Versions | 1.10.0
Video player | 1.6.0
Viewer | 1.2.0

#### Browser log
```
[ERROR] calendar: too much recursion 
Object { app: "calendar", uid: "authenticated:<my_username>" }
calendar.js:2:1765402
    value https://<NC_hostname>/apps/calendar/js/calendar.js?v=772f0140-0:2
    value https://<NC_hostname>/apps/calendar/js/calendar.js?v=772f0140-0:2
    Ul https://<NC_hostname>/apps/calendar/js/calendar.js?v=772f0140-0:2
    n https://<NC_hostname>/apps/calendar/js/calendar.js?v=772f0140-0:2
    c https://<NC_hostname>/apps/calendar/js/calendar.js?v=772f0140-0:2
    _invoke https://<NC_hostname>/apps/calendar/js/calendar.js?v=772f0140-0:2
    t https://<NC_hostname>/apps/calendar/js/calendar.js?v=772f0140-0:2
    Ql https://<NC_hostname>/apps/calendar/js/calendar.js?v=772f0140-0:2
    a https://<NC_hostname>/apps/calendar/js/calendar.js?v=772f0140-0:2
    N https://<NC_hostname>/apps/calendar/js/calendar.js?v=772f0140-0:2
    N https://<NC_hostname>/apps/calendar/js/calendar.js?v=772f0140-0:2
    l https://<NC_hostname>/apps/calendar/js/calendar.js?v=772f0140-0:2
```

**Comment by georgehrke** [comment]
> The calendar is a public one, so I have attached it to the issue.

Where did you attach it? :)

**Comment by albatros69** [comment]
> Where did you attach it? :)
Now it is (sorry)
[Calendrier_CB.ics.txt](https://github.com/nextcloud/calendar/files/4108412/Calendrier_CB.ics.txt)

**Comment by georgehrke** [comment]
@albatros69 I imported the calendar you provided without any issues.
Can you please check https://github.com/nextcloud/calendar/issues/1843#issuecomment-575996362

In Calendar 2.0, we changed the way subscriptions work. Please make sure that cron is properly configured and oc_jobs and oc_calendarobjects is properly filled as described in that ticket.

**Comment by albatros69** [comment]
Cron is properly configured to tun every 5 minutes. I ran it by hand, without any issue (no error in the logs).
I checked the database as per #1843, and I can find some entries:
```
nextclouddb=# select count(*) from oc_calendarobjects where calendarid=3 and calendartype=1;
 count 
-------
   214
(1 row)
```
Moreover, `oc_jobs` is mentioning a `OCA\DAV\BackgroundJob\RefreshWebcalJob` about this particular subscription. However, I now realise that the other subscriptions also have the same issue (too much recursion), but are displaying events (which lured me). However, they are not updated either...
